Library/ViewModels/DeprecatedRewardPledgeViewModel.swift

```diff
@@ -1006,7 +1006,7 @@ private func paymentSummaryItems(forProject project: Project,
   return paymentSummaryItems
 }
 
-private func defaultShippingRule(fromShippingRules shippingRules: [ShippingRule]) -> ShippingRule? {
+internal func defaultShippingRule(fromShippingRules shippingRules: [ShippingRule]) -> ShippingRule? {
```

Comment:
Since this is now shared between new and deprecated code should we maybe extract it to `SharedFunctions` or other helper file in `KsApi`? Would also allow us to test it since it's now not `private` function

Reply:
The intention is to move this function to be `private` in `PledgeViewModel` once we delete `DeprecatedRewardPledgeViewModel`. It's currently behaving as if it were `private` to both the `PledgeViewModel` and the `DeprecatedRewardPledgeViewModel`, and it's functionality is tested twice, both in `DeprecatedRewardPledgeViewModelTests` and `PledgeViewModelTests`. We should leave it as is to minimize the amount of cleanup work when we remove the deprecated files.